layouts/partials/blog/summary.html

```diff
@@ -6,7 +6,7 @@
     {{ .Summary }}
 </p>
 <p>
-    <a href="{{ .Permalink }}">
+    <a data-track="read-more" href="{{ .Permalink }}">
```

Comment:
Similarly, should the convention be `read-more` or `[kind-of-thing]-read-more`? Is just `read more` enough, given we aren't relying on DOM positioning? It may be, since with the href, we'll probably be able to pinpoint the source click, but this should be clarified somewhere.